src/help.sh

```diff
@@ -7,36 +7,38 @@ function kworkflow_help()
     '' \
     'The current supported targets are:' \
     '  Host - this machine' \
-    '  Qemu - qemu machine' \
     '  Remote - machine reachable via the network' \
     '' \
     'Commands:' \
-    '  backup - Save or restore kw data' \
-    '  bd - Build and install modules' \
+    '  init - Initialize kworkflow config file' \
     '  build,b - Build kernel' \
-    '  clear-cache - Clear files generated by kw' \
-    '  codestyle,c - Apply checkpatch on directory or file' \
-    '  config,g - Set kw config options' \
-    '  env - Handle kw envs' \
-    '  debug - Linux kernel debug utilities' \
     '  deploy,d - Deploy a new kernel image to a target machine' \
-    '  device - Show basic hardware information' \
+    '  bd - Build and install modules' \
```

Comment:
Isn't it more than just building and installing modules? Sorry if I'm mixing things up.